packages/ui/src/components/vuestic-mixins/ColorMixin.ts

```diff
@@ -14,7 +14,7 @@ export const isCssColor = (strColor: string): boolean => {
 }
 
 export const useColor = () => {
-  const { getTheme = () => ({}) } = setupColors() as any || {}
+  const { getTheme = () => ({}) } = useColors() as any || {}
```

Comment:
`= () => ({})` and `as any || {}` seems redundant to me.

Also what's `getTheme`?